packages/grid/x-data-grid-premium/src/hooks/features/clipboard/useGridClipboardImport.ts

```diff
@@ -0,0 +1,404 @@
+import * as React from 'react';
```

Comment:
Although it won't work with sorting/filtering - the empty rows added to the grid might not necessarily appear at the very end 🤨

Reply:
Actually, we don't need to revert the paste behavior as long as row selection on click is disabled, here's a demo:
https://codesandbox.io/s/cocky-khorana-ssu3ef?file=/demo.tsx

But it still won't work with sorting/filtering.

Reply:
> Awesome 🎉 IMO, this might be useful in some use-cases, maybe add a small demo in docs?

I would avoid adding a demo for it right now, mostly because it doesn't work with sorting/filtering applied.
I think to fully support this use case, we will have to provide an API to add a row to the end while ignoring sorting/filtering.

> Would considering an external cell as an anchor (from where we start pasting) if it's outside the selected rows make the UX a bit better in your opinion? 

I think this could be useful particularly when `checkboxSelection` is enabled, since rows are not deselected automatically in this case.
It will also make the paste resolution logic even more complex (and I already struggle to explain it in the docs 😅).
I would wait for some feedback first, before actually changing it.